ui/lib/replication/addon/controllers/application.js

```diff
@@ -121,25 +121,13 @@ export default Controller.extend(copy(DEFAULTS, true), {
     onSubmit(/*action, mode, data, event*/) {
       return this.submitHandler(...arguments);
     },
-    copyClose(successMessage) {
-      // separate action for copy & close button so it does not try and use execCommand to copy token to clipboard
-      if (!!successMessage && typeof successMessage === 'string') {
-        this.flashMessages.success(successMessage);
-      }
+    closeTokenModal() {
       this.toggleProperty('isModalActive');
       this.transitionToRoute('mode.secondaries');
+      this.set('isTokenCopied', false);
     },
-    toggleModal(successMessage) {
-      if (!!successMessage && typeof successMessage === 'string') {
-        this.flashMessages.success(successMessage);
-      }
-      // use copy browser extension to copy token if you close the modal by clicking outside of it.
-      const htmlSelectedToken = document.querySelector('#token-textarea');
-      htmlSelectedToken.select();
-      document.execCommand('copy');
-
-      this.toggleProperty('isModalActive');
-      this.transitionToRoute('mode.secondaries');
```

Comment:
this method wasn't working, probably because `execCommand` was deprecated [[docs](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand)]